hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -470,7 +517,28 @@ export class Datanodes extends React.Component<Record<string, object>, IDatanode
         </div>
 
         <div className='content-div'>
+          {totalCount > 0 &&
+            <div style={{ marginBottom: 16 }}>
+              <Popconfirm
+                disabled={!hasSelected}
+                placement="topLeft"
+                title={`Are you sure you want Recon to stop tracking these selected ${selectedRowKeys.length} datanodes ?`}
```

Comment:
```suggestion
                title={`Are you sure you want Recon to stop tracking the selected ${selectedRowKeys.length} datanodes?`}
```

Reply:
@dombizita  Done Changes in latest commit.